src/H5FDmpio.c

```diff
@@ -133,23 +133,26 @@ static const H5FD_class_mpi_t H5FD_mpio_g = {
 };
 
 #ifdef H5FDmpio_DEBUG
-/* Flags to control debug actions in H5Fmpio.
- * Meant to be indexed by characters.
- *
- * 'c' show result of MPI_Get_count after read
```

Comment:
Was the 'c' flag never implemented? Seems like something that would actually be helpful when debugging certain issues like reading/writing > 2GB where diagnosing failure from overflow on the byte count.

Reply:
I don't believe that 'c' was ever implemented - I don't recall ever seeing it.

Reply:
Ok - might be something worth pursuing in the future then.